mod_ngarn/__init__.py

```diff
@@ -27,11 +27,14 @@ def script():
     default=os.getenv("DBTABLE", "public.modngarn_job"),
 )
 @click.option("--limit", default=300, help="Limit jobs (Default: 300)")
-def run(queue_table, limit):
+@click.option("--max-delay", help="Max delay for failed jobs (seconds)(Default: None)")
+def run(queue_table, limit, max_delay):
```

Comment:
run(queue_table, limit=300, max_delay=None) 

Is that better? You don't have to put in None everywhere.

Reply:
This is from lib that I use for produce the cmdline, which already handle the default values

e.g. `@click.option("--limit", default=300, help="Limit jobs (Default: 300)")`

For `max_delay`, it already defaults to None.

I have to put in None everywhere else because I want the actual code to be more strict.

Reply:
but shouldn't it go the same way with click.option if called directly?